csharp/ql/lib/semmle/code/csharp/frameworks/system/web/ui/WebControls.qll

```diff
@@ -2,6 +2,7 @@
 
 import csharp
 private import semmle.code.csharp.frameworks.system.web.UI
+private import semmle.code.csharp.dataflow.ExternalFlow
```

Comment:
Add a `private import semmle.code.csharp.frameworks.system.web.ui.WebControls` to the module `Frameworks` in `ExternalFlow.qll` to ensure bidirectional importing.